src/deluge/storage/flash_storage.cpp

```diff
@@ -636,6 +641,13 @@ void readSettings() {
 		gui::menu_item::fillColourMenu.value = gui::menu_item::Colour::AMBER;
 		gui::menu_item::onceColourMenu.value = gui::menu_item::Colour::MAGENTA;
 	}
+	// defaultSliceMode = static_cast<SampleRepeatMode>(buffer[167]);
+	if (buffer[167] >= kNumRepeatModes) {
```

Comment:
you can remove the commented out line 644

Reply:
alright